src/System.Memory/src/System/Buffers/ReadOnlySequence.Helpers.cs

```diff
@@ -154,6 +154,84 @@ private ReadOnlyMemory<T> GetFirstBufferSlow(object startObject, bool isMultiSeg
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private ReadOnlySpan<T> GetFirstSpan()
+        {
+            object startObject = _startObject;
+
+            if (startObject == null)
+                return default;
+
+            int startIndex = _startInteger;
+            int endIndex = _endInteger;
+
+            bool isMultiSegment = startObject != _endObject;
+
```

Comment:
All these comments are duplicated from GetFirst() method which returns ReadOnlyMemory instead of ReadOnlySpan